Library/Homebrew/brew.sh

```diff
@@ -427,14 +436,16 @@ Your curl executable: $(type -p $HOMEBREW_CURL)"
 Minimum required version: ${HOMEBREW_MINIMUM_GIT_VERSION}
```

Comment:
`Please update your system Git.` above should only refer to `system curl` if `HOMEBREW_GIT_PATH` wasn't set by the user.